thirdparty/runtime_environments.go

```diff
@@ -19,6 +19,10 @@ const (
 	ToolchainsType = "Toolchains"
 	OSType         = "OS"
 
+	OSNameField      = "PRETTY_NAME"
+	OSKernelField    = "Kernel"
```

Comment:
is there a reason this is not capitalized but everything else is

Reply:
That is how the runtime environments API shows the fields.